static/js/dev.ts

```diff
@@ -44,6 +49,11 @@ window.onload = function () {
     chartElem.style.width = width + "px";
     return containerId;
   }
+  let width_menu = 600
```

Comment:
const widthMenu = 600;

Try to use const as much as possible; js style guide uses camelCase for variable and functions

Reply:
Got it. Thanks.